api/article/views.py

```diff
@@ -7,8 +9,23 @@ class ArticleViewSet(viewsets.ModelViewSet):
     serializer_class = ArticleSerializer
```

Comment:
Do you still need that?

Reply:
Wait a minute. Why are you using `serializer_class_map` at all?

Reply:
OMG ))))))))))))))))))